### PR TITLE
Feat/add know universe description

### PR DESCRIPTION
**Pull request by SonnevilleDamien**
In this PR :
- Modification of the name of the final dataset Sirene 
- add the characteristic of the final dataset 
- add the description of the data space know universe 
- add the know universe datasets
- Add 2 "knowledge datasets" in the dataset.json file for the Know Universe 
 - Place the service "Data Selector" in first position in the service.json file
- Add a service in second position of the service.json file for Know Universe

### CODE REVIEW

**Review by RaphaelEscrig — APPROVED** [review]

**Review by ccamel — APPROVED** [review]
Thx :relieved:

File: data/dataset.json [hunk 1]
```diff
@@ -219,17 +221,111 @@
     "dataspaceId": "ef347285-e52a-430d-9679-dcb76b962ce7",
     "mainPicture": "https://images.unsplash.com/photo-1537721664796-76f77222a5d0",
     "type": "dataset",
-    "name": "BASE SIRÈNE - Exploitation agricole",
+    "name": "BASE SIRENE - agriculture sylviculture pêche",
     "access": "PUBLIC",
-    "categories": ["open data", "INSEE", "exploitation agricole"],
-    "description": "Ce je de donnée référence toutes les entreprises du secteur agricole en france en activité. Obtenu grâce à un enchainement de traitements de la donnée à l'aide du protocole OKP4.",
+    "categories": ["open data", "INSEE", "agriculture", "sylviculture", "pêche"],
+    "description": "Ce je de donnée référence toutes les entreprises de l'agriculture, de la sylviculture et de la pêche en France en activité. Obtenu grâce à un enchainement de traitements de la donnée à l'aide du protocole OKP4.",
```
Comment (fredericvilcot) [inline data/dataset.json:227]:
```suggestion
    "description": "Ce jeu de données référence toutes les entreprises de l'agriculture, de la sylviculture et de la pêche en France en activité. Obtenu grâce à un enchainement de traitements de la donnée à l'aide du protocole OKP4.",
```

File: data/dataset.json [hunk 2]
```diff
@@ -219,17 +221,111 @@
...
+  },
+  {
+    "id": "86568354-fa42-47e4-86f5-ae8c31a26e67",
+    "dataspaceId": "ti847285-e52a-430d-9679-dcb76b962ce7",
+    "mainPicture": "https://okp4.network/wp-content/themes/okp4/img/know.png",
+    "type": "dataset",
+    "name": "Cosmos Hub transactions",
+    "access": "PUBLIC",
+    "categories": ["cosmos", "blockchain", "open data"],
+    "description": "Numia Data vise à fournir les données de l'univers Cosmos comme un bien public. Bien que les données de la blockchain soient généralement publiques par conception, il n'est pas très facile d'interroger, de lire et d'analyser les données directement à partir de la chaîne. Numia corrige ce problème. Ce jeu de donnée référence les transactions de la blockchain Cosmos Hub.",
```
Comment (fredericvilcot) [inline data/dataset.json:247]:
```suggestion
    "description": "Numia Data vise à fournir les données de l'univers Cosmos comme un bien public. Bien que les données de la blockchain soient généralement publiques par conception, il n'est pas très facile d'interroger, de lire et d'analyser les données directement à partir de la chaîne. Numia corrige ce problème. Ce jeu de données référence les transactions de la blockchain Cosmos Hub.",
```

Comment (MdechampG) [inline data/dataset.json:247]: données

File: data/dataset.json [hunk 3]
```diff
@@ -219,17 +221,111 @@
...
+  },
+  {
+    "id": "6e5d7a1b-0cf4-4466-b678-878d55bcad74",
+    "dataspaceId": "ti847285-e52a-430d-9679-dcb76b962ce7",
+    "mainPicture": "https://okp4.network/wp-content/themes/okp4/img/know.png",
+    "type": "dataset",
+    "name": "Evmos transactions",
+    "access": "PUBLIC",
+    "categories": ["evmos", "blockchain", "open data"],
+    "description": "Numia Data vise à fournir les données de l'univers Cosmos comme un bien public. Bien que les données de la blockchain soient généralement publiques par conception, il n'est pas très facile d'interroger, de lire et d'analyser les données directement à partir de la chaîne. Numia corrige ce problème. Ce jeu de donnée référence les transactions de la blockchain Evmos.",
```
Comment (fredericvilcot) [inline data/dataset.json:265]:
```suggestion
    "description": "Numia Data vise à fournir les données de l'univers Cosmos comme un bien public. Bien que les données de la blockchain soient généralement publiques par conception, il n'est pas très facile d'interroger, de lire et d'analyser les données directement à partir de la chaîne. Numia corrige ce problème. Ce jeu de données référence les transactions de la blockchain Evmos.",
```

File: data/dataset.json [hunk 4]
```diff
@@ -219,17 +221,111 @@
...
+  },
+  {
+    "id": "eae97137-62ef-4398-939c-438fd6dce0d0",
+    "dataspaceId": "ti847285-e52a-430d-9679-dcb76b962ce7",
+    "mainPicture": "https://okp4.network/wp-content/themes/okp4/img/know.png",
+    "type": "dataset",
+    "name": "Osmosis transactions",
+    "access": "PUBLIC",
+    "categories": ["osmosis", "blockchain", "open data"],
+    "description": "Numia Data vise à fournir les données de l'univers Cosmos comme un bien public. Bien que les données de la blockchain soient généralement publiques par conception, il n'est pas très facile d'interroger, de lire et d'analyser les données directement à partir de la chaîne. Numia corrige ce problème. Ce jeu de donnée référence les transactions de la blockchain Osmosis.",
```
Comment (fredericvilcot) [inline data/dataset.json:283]:
```suggestion
    "description": "Numia Data vise à fournir les données de l'univers Cosmos comme un bien public. Bien que les données de la blockchain soient généralement publiques par conception, il n'est pas très facile d'interroger, de lire et d'analyser les données directement à partir de la chaîne. Numia corrige ce problème. Ce jeu de données référence les transactions de la blockchain Osmosis.",
```

File: data/dataset.json [hunk 5]
```diff
@@ -219,17 +221,111 @@
...
+  },
+  {
+    "id": "1cd663d4-07b6-40e2-a59d-6e7c455fe7ed",
+    "dataspaceId": "ti847285-e52a-430d-9679-dcb76b962ce7",
+    "mainPicture": "https://okp4.network/wp-content/themes/okp4/img/know.png",
+    "type": "dataset",
+    "name": "Cosmos Hub transactions labelled",
+    "access": "PUBLIC",
+    "categories": ["cosmos", "blockchain", "opk4"],
+    "description": "Ce jeu de donnée référence les transactions de la blockchain Cosmos Hub où les adresses des validateurs ont été labellisés à l'aide d'un service du protocole OKP4.",
```
Comment (MdechampG) [inline data/dataset.json:301]: labellisées

Comment (MdechampG) [reply data/dataset.json:301]: données

File: data/dataset.json [hunk 6]
```diff
@@ -219,17 +221,111 @@
...
+  },
+  {
+    "id": "0d513ee9-f861-47ba-a021-4510ba303f7b",
+    "dataspaceId": "ti847285-e52a-430d-9679-dcb76b962ce7",
+    "mainPicture": "https://okp4.network/wp-content/themes/okp4/img/know.png",
+    "type": "dataset",
+    "name": "Evmos transactions labelled",
+    "access": "PUBLIC",
+    "categories": ["evmos", "blockchain", "okp4"],
+    "description": "Ce jeu de donnée référence les transactions de la blockchain evmos où les adresses des validateurs ont été labellisés à l'aide d'un service du protocole OKP4.",
```
Comment (fredericvilcot) [inline data/dataset.json:320]:
```suggestion
    "description": "Ce jeu de données référence les transactions de la blockchain evmos où les adresses des validateurs ont été labellisées à l'aide d'un service du protocole OKP4.",
```

File: data/dataspace.json [hunk 7]
```diff
@@ -17,10 +17,10 @@
     "name": "Know Universe",
     "access": "PUBLIC",
     "creator": "OKP4",
-    "categories": ["agriculture", "open data", "demonstrateur", "dataviz"],
-    "description": "estibulum pharetra, magna eget imperdiet fringilla, metus orci sagittis elit, blandit pretium felis sem sed neque. Etiam commodo nisi lorem, non fringilla tellus euismod at.",
+    "categories": ["cosmos", "open data", "blockchain", "OKP4"],
+    "description": " Le Data Space Know Universe vise à valoriser des données financières inhérentes à l'économie du jeton $KNOW. Ces données peuvent par exemple être partagées par l'un des 150 validateurs de la blockchain OKP4",
```
Comment (MdechampG) [inline data/dataspace.json:21]: espace et point

**Review by fredericvilcot — CHANGES_REQUESTED** [review]

File: data/service.json [hunk 8]
```diff
@@ -1,4 +1,38 @@
...
+    "nbOutputFiles": 1,
+    "outputFormat": ["csv"],
+    "createdOn": "2022-09-28T11:13:13Z",
+    "updatedOn": "2022-09-28T11:13:13Z"
+    },
+    {
+      "id": "2e0450b4-3f62-4bdf-860e-cc5462b610c7",
+      "dataspaceId": "eti847285-e52a-430d-9679-dcb76b962ce7",
+      "type": "service",
+      "name": "Cosmos Validator Labaliser",
```
Comment (MdechampG) [inline data/service.json:23]: labeller

File: data/service.json [hunk 9]
```diff
@@ -1,4 +1,38 @@
...
+      "access": "PUBLIC",
+      "categories": ["traitement de données", "labellisation"],
+      "description": "L'outil Cosmos Validator Labaliser permet de labelliser les adresses des validateurs de l'univers Cosmos.",
```
Comment (MdechampG) [inline data/service.json:26]: labeller

**Review by fredericvilcot — APPROVED** [review]
Great, thx!

**Review by ErikssonJoakim — APPROVED** [review]

**Review by ccamel — APPROVED** [review]

**Review by RaphaelEscrig — APPROVED** [review]

**Review by MdechampG — APPROVED** [review]

**Pull request merged**